src/light_elf.h

```diff
@@ -26,7 +26,9 @@
 
 #include <inttypes.h>
 
-__BEGIN_DECLS
+#ifdef _cplusplus
```

Comment:
Doesn't it have to be `__cplusplus`? See e.g., https://renenyffenegger.ch/notes/development/languages/C-C-plus-plus/preprocessor/macros/predefined/__cplusplus.

Reply:
fixed